fdbserver/DataDistributionQueue.actor.cpp

```diff
@@ -912,10 +933,10 @@ ACTOR Future<Void> dataDistributionRelocator( DDQueueData *self, RelocateData rd
 				bestTeams.clear();
 				while( tciIndex < self->teamCollections.size() ) {
 					double inflightPenalty = SERVER_KNOBS->INFLIGHT_PENALTY_HEALTHY;
-					if(rd.priority >= PRIORITY_TEAM_UNHEALTHY) inflightPenalty = SERVER_KNOBS->INFLIGHT_PENALTY_UNHEALTHY;
-					if(rd.priority >= PRIORITY_TEAM_1_LEFT) inflightPenalty = SERVER_KNOBS->INFLIGHT_PENALTY_ONE_LEFT;
+					if(rd.healthPriority == SERVER_KNOBS->PRIORITY_TEAM_UNHEALTHY || rd.healthPriority == SERVER_KNOBS->PRIORITY_TEAM_2_LEFT) inflightPenalty = SERVER_KNOBS->INFLIGHT_PENALTY_UNHEALTHY;
+					if(rd.healthPriority == SERVER_KNOBS->PRIORITY_TEAM_1_LEFT || rd.healthPriority == SERVER_KNOBS->PRIORITY_TEAM_0_LEFT) inflightPenalty = SERVER_KNOBS->INFLIGHT_PENALTY_ONE_LEFT;
```

Comment:
Why don't we add `inflightPenalty ` for split and merge request?
Won't those two requests add extra `loadBytes` to the source servers?

Reply:
An `inflightPenalty` larger than one is telling the system that it is better to prioritize completing the movement fast over putting it in a good location. This means this system will have more work to do later, because it could lead to an unbalanced bytes stored across storage servers.

The reason we do this for unhealthy relocations is because if you have a few newly added servers without a lot of bytes, and then exclude a different server, it would try and move everything from the excluded server to the newly added servers.

This same overload does not exist for splits and merges.